docs/how_to_run.md

```diff
@@ -1,31 +1,35 @@
-In env vars:
+In env vars (.env file at the project root):
 
     - replace the ip adress in EUREKA_SERVER_URL with your own
-    - fill in the SECRET_KEY and GOOGLE_CLIENT_SECRET with the secrets
+    - fill in the SECRET_KEY and GOOGLE_CLIENT_SECRET, and API_KEY with the secrets
 
 # Development
-python env.py --env dev
-./gradlew makeSharedObjs
-./gradlew service:build
-docker network create netw
-cd service
-docker-compose -f docker-compose.dev.yml up --build
+Clone the repository and change directory to the project root  
+Run env.py to create .env files for each microservice ```python env.py --env dev```    
+Run ```./gradlew clean``` (if necessary)  
+Run ```./gradlew makeSharedObjs```  
+Run ```./gradlew build```  
+Go to the EurekaService ```cd .\EurekaService\```  
+Run ```docker compose -f docker-compose.dev.yml up``` to start the Eureka server
+Change directory to the service you want to run ```cd ..\service``` (replace service with the path to the service)  
+Run ```docker compose -f docker-compose.dev.yml up --build``` to build the Docker image and start the service  
 
 # Production
-git clone https://github.com/s1m0nWasTaken/CS4337_BigDataProject.git
-cd CS4337_BigDataProject
-python3 env.py --env prod
-./gradlew makeSharedObjs
-./gradlew ${service}:build
-cd ${service}
-sudo docker compose -f docker-compose.prod.yml up --build
+Clone the repository and change directory to the project root  
+Run env.py to create .env files for each microservice ```python env.py --env prod```    
+Run ```./gradlew clean``` (if necessary)  
+Run ```./gradlew makeSharedObjs```  
+Run ```./gradlew build```  
+Go to the EurekaService ```cd .\EurekaService\```  
```

Comment:
Probably could leave prod to be just how to start a service rather than specifically saying Eureka first as it doesn't need to be first